javascript/features/gangs/gang_commands.js

```diff
@@ -53,6 +54,18 @@ const GOAL_QUESTION = {
     }
 };
 
+// Options for asking the player what the gang's skin should be.
+const SKIN_QUESTION = {
+    question: 'Choose your gang\'s skin',
+    message: 'What is the skin your gang will use?',
+    constraints: {
+        validation: /^([0-9]|[1-9][0-9]|1[0-1][0-9]|120|12[2-9]|1[3-9][3-9]|2[0-9][0-9])$/u,
```

Comment:
Thanks for excluding 121 :).

I wonder if we should have a slightly more rigid verification, as e.g. skins 300-311 are valid too in the latest versions of SA-MP. We might not want every pick to be selectable though, the `ClassManager` in Pawn has some logic there.

Reply:
I updated the regex to read it from the JSON.